index.html

```diff
@@ -2290,7 +2326,7 @@ <h4 id="multibase">The `multibase` Datatype</h3>
             <dd>
 Any element of the lexical space is mapped to the value space by base-decoding
 the value based on the base-decoding alphabet associated with the
-first  [[?MULTIBASE]] character in the lexical string.
+first <a href="#multibase">Multibase</a> character in the lexical string.
```

Comment:
Again, "multibase character" is currently undefined.

Reply:
It's defined here:

https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#multibase-0

What else are you looking for?